doc/modules/qualitymetrics/firing_range.rst

```diff
@@ -0,0 +1,40 @@
+Firing range (:code:`firing_range`)
+===================================
+
+
+Calculation
+-----------
+
+The firing range indicates the dispersion of the firing rate of a unit across the recording. It is computed by
+taking the difference between the 95-th and 5th percentiles firing rates computed over short time bins (e.g. 10 s).
```

Comment:
```suggestion
taking the difference between the 95th percentile's firing rate and the 5th percentile's firing rate computed over short time bins (e.g. 10 s).
```

Reply:
I think making this slightly more explicit will read cleaner. Otherwise it technically needs a percentiles' apostrophe for a plural possessive. Which is easier to confuse a reader.